go.mod

```diff
@@ -38,14 +38,15 @@ replace (
 )
 
 require (
-	github.com/aws/aws-sdk-go v1.25.48
+	github.com/aws/aws-sdk-go v1.27.0
```

Comment:
Is this change needed?

Reply:
`go mod tidy` did that..things like this will happen unless the version is pinned with a replace directive. In general though, Go modules are pretty good though at not introducing breakages with these unwarranted updates.